pkg/serverless/trigger/events.go

```diff
@@ -153,7 +153,7 @@ func Unmarshal(payload []byte) (map[string]interface{}, error) {
 }
 
 func isAPIGatewayEvent(event map[string]interface{}) bool {
-	return json.GetNestedValue(event, "requestcontext", "stage") != nil &&
+	return json.GetNestedValue(event, "requestcontext") != nil &&
```

Comment:
Two things:

1. Is the requestcontext guaranteed to be in the payload?
2. and is that combination of requestcontext, httpmethod, and resource unique? If not, other payloads will be mistakenly classified as APIGateway